pkg/shell/superuser.jsx

```diff
@@ -180,6 +187,32 @@ const UnlockDialog = ({ proxy, host }) => {
             <Button variant="secondary" className='btn-cancel' onClick={cancel}>
                 {_("Close")}
             </Button>);
+    } else if (bridges) {
+        const apply = () => {
+            start(bridge);
+        };
+
+        title = _("Switch to administrative access");
+        body = (
+            <Form isHorizontal onSubmit={event => { apply(); event.preventDefault(); return false }}>
+                <FormGroup fieldId="switch-to-admin-access-bridge-select"
+                           label={_("Method")}>
+                    <FormSelect value={bridge} onChange={setBridge} isDisabled={busy}>
+                        { bridges.map(b => <FormSelectOption value={b} key={b} label={b} />) }
+                    </FormSelect>
+                </FormGroup>
+            </Form>);
+
+        footer = (
+            <>
+                <Button variant='primary' onClick={apply} isDisabled={busy} isLoading={busy}>
+                    {_("Authenticate")}
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17536-20220708-114053-02201755-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)